packages/web-js/src/core/Corbado.ts

```diff
@@ -15,10 +15,17 @@ export class Corbado {
     return this.#getCorbadoAppState().user;
   }
 
+  /**
+   * @deprecated Use sessionToken() instead
+   */
   get shortSession() {
     return this.#getCorbadoAppState().shortSession;
   }
 
+  get sessionToken() {
+    return this.#getCorbadoAppState().shortSession;
+  }
+
   get shortSessionChanges() {
```

Comment:
I think we should also deprecate shortSessionChanges then (and add sessionTokenChanges)

Reply:
yes good point, done